src/components/Page-Title/presets/0-default.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PageTitle from '../Page-Title';
+
+export default (
+  <PageTitle
+    uxpId="pagetitle"
+    title="Page Title"
+    helpIcon=""
```

Comment:
Is this correct? Is not the value supposed to be {false} instead ?

Reply:
changes added